functions-and-scope.js

```diff
@@ -82,3 +144,20 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // highestGrade(grades) geeft 9
 // highestGrade([6, 4, 5]) geeft 6
 // highestGrade([8, 9, 4, 6, 10]) geeft 10
+
+function highestGrade (gradesArray) {
+    let highest = 0
```

Comment:
Voor deze opdracht niet heel erg, maar wees je ervan bewust dat als je een array als `[-3, -1, -2, -5]` meegeeft aan deze functie, dat je dan `0` terugkrijgt. Je kan daarom altijd beter de eerste waarde van de array pakken
```suggestion
    let highest = gradesArray[0];
```